tests/integration/test_posts.py

```diff
@@ -51,3 +52,12 @@ def test_post_detail(self, client: FlaskClient, post: Post) -> None:
         assert response.json['body'] == 'ccc'
         assert response.json['createdAt'] == datetime_to_rfc822_string(post.created_at)
         assert response.json['updatedAt'] == datetime_to_rfc822_string(post.updated_at)
+
+    def test_post_detail_not_found(self, client: FlaskClient, post: Post) -> None:
+        response = client.get(f'/api/posts/100')
+        assert response.status_code == 404
+        assert response.json['status'] == 404
+        assert response.json['title'] == 'Post Not Found'
+        assert response.json['timestamp'] == datetime_to_rfc822_string(datetime.now())
```

Comment:
does this work? the `datetime.now()` call in the test might return a different value than the response from the API, right?